package_manifest.yaml

```diff
@@ -66,6 +66,23 @@ copr_projects:
             - "{{ foreman_staging }}/rhel-{{ rhel_8 }}-x86_64"
           comps_file: "{{ inventory_dir }}/comps/comps-foreman-el{{ rhel_8 }}.xml"
           buildroot_packages: "{{ core_buildroot_packages }}"
+    foreman-copr-with-el9:
+      copr_project_name: "foreman-{{ foreman_version }}-staging"
```

Comment:
If this does not change, why add a new entry?

Reply:
how else would we refer to it when a group is configured to use the `with-el9` config?